### ISSUE TITLE
First keypress doesn't work when bind-key is active on emacs 28, emacs daemon + emacs client is used

### ISSUE THREAD

**Issue by kurnevsky** [jwiegley/use-package#973]
Start emacs 28 daemon, then try to edit some file with emacsclient, like `emacsclient -nw ~/file`. After file is opened first keypress will result in `x is undefined` (or quit if you press q). It doesn't happen consistently but always happens after fresh emacs daemon start (so it might be a emacs bug). It also works if you move mouse cursor before typing first letter. Not sure how to investigate it further, so decided to report here.

Emacs version: 28.0.91
OS: Linux
Native compilation is enabled.

**Comment by kurnevsky** [comment]
cc @Eli-Zaretskii

**Comment by Eli-Zaretskii** [comment]
What is `bind-key`?  Are you sure it isn't its fault?

**Comment by kurnevsky** [comment]
bind-key is a part of use-package: https://github.com/jwiegley/use-package/blob/master/bind-key.el
That's why I reported it here. It might be its fault - not sure. But the behavior changed in emacs 28 so I guessed it might be something there as well.

**Comment by kurnevsky** [comment]
@Eli-Zaretskii well, it's not `bind-key` bug after all - I looked through the code and didn't see anything suspicious. I have the same behavior with `(require 'easy-mmode)` so it's almost certainly emacs bug. Should I report it to the mailing list? :)

**Comment by Eli-Zaretskii** [comment]
Yes, please report a bug with the minimal recipe as part of the report.  The recipe should ideally start with `emacs -Q`, or if that is impossible, use the smallest possible init file to reproduce the issue.
